profile/isa_ro_crate.md

```diff
@@ -248,9 +248,10 @@ It is based on [schema.org/ScholarlyArticle](https://schema.org/ScholarlyArticle
 |headline|MUST|Text||
 |author|SHOULD|[schema.org/Person](https://schema.org/Person)||
 |url|SHOULD|URL||
+|identifier|SHOULD|Text or URL or [schema.org/PropertyValue](https://schema.org/PropertyValue)|One or many identifiers for this article like a DOI or PubMedID. Can be of type PropertyValue to indicate the kind of reference.|
```

Comment:
Shouldn't `sameAs` be thrown out in favor of `identifier`?

Reply:
Could it be that you're mixing up Issues/PRs here? I can't follow you.